### ISSUE TITLE
DOM Tests

### ISSUE THREAD

**Issue by HitmanInWis** [chilicat/JsChilicat#8]
Is it possible to use JsChilicat for JS tests that involve DOM manipulations?  I see that there is integration w/envjs, which leads me to think this may be possible, but I dont see anywhere in the Usage section how to leverage an HTML page.

**Comment by chilicat** [comment]
Yes you can work with the DOM in your tests but currently it is not possible to leverage HTML. 
I used it mainly for API tests so far.
